src/actions/printers/reading.js

```diff
@@ -98,6 +106,12 @@ export const getWebcamSnapshot = createHttpAction(
         if (webcams.queue && webcams.queue[id]) {
           const timeoutData = webcams.queue[id];
 
+          // no matter what, we always kill ani potentional timeout if set before setting a new one
+          // this prevents ghost intervals staying in the background
+          if (webcams.queue[id] && webcams.queue[id].timeout) {
+            clearTimeout(webcams.queue[id].timeout);
+          }
```

Comment:
One stickler note: JS `clearTimeout` was cleverly designed to ignore any invalid value. Double check is not necessary.

```suggestion
          if (webcams.queue[id]) {
            clearTimeout(webcams.queue[id].timeout);
          }
```